m4_sessions/1710-inning/group_five.md

```diff
@@ -1,10 +1,28 @@
 Group Five
 
-Session Title: 
+Session Title: Crushing Cold Outreach
+
 Description: Why should students come to this session? 
+Are you terrified of cold outreach? Or do you not even know what it is? We'll be doing a live walk through with volunteers of successfully reaching out to people you don't know. We'll address feelings of rejection, imposter syndrome, expectations and targeting strtategies. 
 
 Session Outcomes: What will students leave with or gain from this session?
+- everyone leaves with one message sent or drafted
+- good understanding of future approach
+- mitigation of anxiety around rejection
+
 
 Session Outline:  Include the flow of the session and the time for each section.
+1. Brief intro of what it is, why it useful
+2. fist to 5 about how comfortable you are with it
```

Comment:
Great check for understanding.  You could also have people line up along a "line" and so you can see the spectrum.  You could then match people with someone who is in the same spot or opposite spot from them